src/@next/Button/ButtonStoriesSyle.ts

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+import { Badge } from '../Badge';
+
+export const StyledBadge = styled(Badge)`
```

Comment:
Do we leave the creation of floating badge to the product squad? 🤔

Reply:
yes @marthayaputra ? don't we want more flexibility?

Reply:
Hmm because this is part of Button stories, one might assume this is part of the buttons? But from what  I see here it's not supposed to be one

Reply:
as discussed in slack, we'll create a separate page for the floating badge